core/metacling/src/TCling.cxx

```diff
@@ -5891,11 +5867,92 @@ Int_t TCling::AutoParse(const char *cls)
    return nHheadersParsed > 0 ? 1 : 0;
 }
 
+static void* LazyFunctionCreatorAutoloadForModule(const std::string& mangled_name,
+      cling::Interpreter *fInterpreter) {
+   using namespace llvm::object;
+   using namespace llvm::sys::path;
+   using namespace llvm::sys::fs;
+
+   R__LOCKGUARD(gInterpreterMutex);
+
+   static bool sFirstRun = true;
+   // We don't want to do directory_iterator several times because it contains HDD access
+   static std::set<std::string> Libraries;
```

Comment:
`Libraries` in ROOT coding convention would be `libraries` (or, as it's static maybe `sLibraries`)?